### ISSUE TITLE
losses_avg

### ISSUE THREAD

**Issue by LLLLLLoki** [XMUNLP/Tagger#19]
请问 您在loss中加入
with tf.variable_scope("losses_avg"):
                loss_moving_avg = tf.get_variable("training_loss",
                                                  initializer=100.0,
                                                  trainable=False)
                lm = loss_moving_avg.assign(loss_moving_avg * 0.9 + loss * 0.1)
                tf.summary.scalar("loss_avg/total_loss", lm)

                with tf.control_dependencies([lm]):
                    loss = tf.identity(loss)

为什么要加入一个losses_avg
